lizmap/modules/lizmap/lib/App/JelixContext.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * app context informations inside a Jelix environment
+ *
+ * @author    3liz
+ * @copyright 2020 3liz
+ *
+ * @see      https://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+namespace Lizmap\App;
+
+class JelixContext implements AppContextInterface
+{
+    /**
+     * call and return jApp::config.
+     */
+    public function appConfig()
+    {
+        return \jApp::config();
+    }
+
+    /**
+     * says if the current user has the given right. Call jAcl2::check().
+     *
+     * @param string $right the key of the right to check
+     * @param string $resource the id of a resource if any
+     * @return bool The result of jAcl2::check()
+     */
+    public function aclCheck($role, $resource = null)
+    {
+        return \jAcl2::check($role);
```

Comment:
```suggestion
        return \jAcl2::check($role, $resource);
```

We sould send the resource to jAcl2::check()